hacks/print-etcd-cheatsheet.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
```

Comment:
drop `.sh` suffix for this file, otherwise you can't just run `print-etcd-cheatsheet`

Reply:
When creating the image, [this line](https://github.com/gardener/ops-toolbelt/pull/100/files#diff-585e09338be8d61489c23232e5fe922dae7a54726fcbc6ae95e242750ecdbdcaR123) copies the content of this file to `print-etcd-cheatsheet`. 
Will this not be sufficient? It is then possible to run `print-etcd-cheatsheet` within the image

Reply:
There is a previous copy operation which copies all `./hacks` files to the `/hacks` dir on the container: https://github.com/gardener/ops-toolbelt/blob/f04ab43a645098a45eb6a8768d6e8130b9f45e52/dockerfile-configs/common-components.yaml#L102-L105
So this file will be available twice (this is also the case for `install_etcdctl`). However, it will be available with the `.sh` suffix in the `/hacks` directory.

Reply:
Fair enough 
dropped `.sh` suffix for `print-etcd-cheatsheet`